### PR TITLE
Upstream: Pull in changes from Hugo v0.79.0

### PR DESCRIPTION
**Pull request by felicianotech**

### CODE REVIEW

File: examples/blog/layouts/partials/meta.html
Comment (chayev) [inline examples/blog/layouts/partials/meta.html:5]: We should update this to say Gotham instead.

File: src/hugolib/config.go
Comment (chayev) [inline src/hugolib/config.go:177]: Does this need to be changed as well?

**Review by chayev — COMMENTED** [review]
Added a few comments. All looks good. 

The comments might need to be addressed outside of this PR, so no explicitly requesting changes.

File: src/hugolib/config.go
Comment (felicianotech) [reply src/hugolib/config.go:177]: Not in v1. That would break backwards compatibility.

File: examples/blog/layouts/partials/meta.html
Comment (felicianotech) [reply examples/blog/layouts/partials/meta.html:5]: Why?

**Comment by felicianotech** [comment]
> so no explicitly requesting changes.

@chayev But you didn't approve either so it's still waiting for an approval.

File: examples/blog/layouts/partials/meta.html
Comment (chayev) [reply examples/blog/layouts/partials/meta.html:5]: Our users would see "A Hugo User" there. Your call, we should review this outside of this PR, as this has been like that before this update.

**Review by chayev — APPROVED** [review]
LG!

File: examples/blog/layouts/partials/meta.html
Comment (felicianotech) [reply examples/blog/layouts/partials/meta.html:5]: I opened this back up. I question if users would even see this. I thought it was more of like a test, though it's not named that. Now I'm not sure.

**Pull request merged**
